CMakeLists.txt

```diff
@@ -1,114 +1,58 @@
 
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.1)
 project(clpeak)
 
 if (NOT CMAKE_BUILD_TYPE)
   message(STATUS "Setting build type to Release")
   set(CMAKE_BUILD_TYPE "Release")
 endif()
 
-# Determine machine bitness
-if(CMAKE_SIZEOF_VOID_P EQUAL 8)
-  set(BITNESS 64)
-else()
-  set(BITNESS 32)
+# FindOpenCL.cmake doesn't use OPENCL_ROOT so force it to copy OPENCL_ROOT to
+# AMDAPPSDKROOT which it does use
+if(OPENCL_ROOT)
+  set(ENV{AMDAPPSDKROOT} ${OPENCL_ROOT})
```

Comment:
I am not entirely happy with this bit but I can't see a way to point the current FindOpenCL at POCL without it.